std/math/bits/conversion_binary.go

```diff
@@ -54,7 +53,7 @@ func fromBinary(api frontend.API, digits []frontend.Variable, opts ...BaseConver
 func toBinary(api frontend.API, v frontend.Variable, opts ...BaseConversionOption) []frontend.Variable {
 	// parse options
 	cfg := baseConversionConfig{
-		NbDigits:             api.Compiler().Curve().Info().Fr.Bits,
+		NbDigits:             api.Compiler().Field().BitLen(),
```

Comment:
If implement `BitLen()` in interface (described in other comment), then can replace.